src/couch_mrview/src/couch_mrview_http.erl

```diff
@@ -582,9 +582,6 @@ parse_param(Key, Val, Args, IsDecoded) ->
             Args#mrargs{callback=couch_util:to_binary(Val)};
         "sorted" ->
             Args#mrargs{sorted=parse_boolean(Val)};
-        "partition" ->
```

Comment:
are we firmly against ?partition= working against _view? I quite like it for its simplicity.

Reply:
I think we against it otherwise its going to get really confusing having a ?partition and a new url

Reply:
I would rather keep it. It's a feature of the map/reduce code and should be exposed there, rather than only reachable obscurely through mango.